README.md

```diff
@@ -92,15 +92,16 @@ The tfschema requires the provider's dependency library version to:
 
 # Rules of finding provider's binary
 When `terraform init` command is executed, provider's binary is installed under the auto installed directory ( .terraform/plugins/`<SOURCE ADDRESS>`/`<VERSION>`/`<OS>_<ARCH>` ) by default.
-The tfschema can use the same provider's binary as terraform uses, so you can run `tfschema` command in the same directory where you run the `terraform` command.
+The tfschema can use the same provider's binary as terraform uses, so you can run `tfschema` command in the same directory where you run the `terraform` command
+Alternatively, you can set the environment variable `$TFSCHEMA_ROOT_DIRECTORY` to be this directory. If `$TFSCHEMA_ROOT_DIRECTORY` is not set, it will default to the current directory
```

Comment:
I prefer `TFSCHEMA_ROOT_DIR` to `TFSCHEMA_ROOT_DIRECTORY` for simplicity.

Reply:
Changed in bce333cbaddbaea64923a674569f377182ca655c